test/test_logs.py

```diff
@@ -0,0 +1,36 @@
+from base_test import BaseTestCase
+from util import rand_str, rand_email
+
+
+class TestLogs(BaseTestCase):
+
+    def test_custom_audit_log(self):
+        # create a test user
+        email = rand_email()
+        user = self.uk.users.create_user(email=email, password=rand_str(14))
+        # create a test audit log
+        log = self.uk.logs.log_event(actor=user.id, actor_ip='::1', actor_useragent='TestLogs', actee=user.id,
+                                     action='test.custom_audit_log', details='Testing custom audit logs')
+        self.assertEqual(log.actor, user.id)
+        self.assertEqual(log.actor_ip, '::1')
+        self.assertEqual(log.actor_useragent, 'TestLogs')
+        self.assertEqual(log.actee, user.id)
+        self.assertEqual(log.action, 'test.custom_audit_log')
+        self.assertEqual(log.details, 'Testing custom audit logs')
+
+    def test_custom_audit_log_invalid_actor(self):
+        # create a test user
+        email = rand_email()
+        user = self.uk.users.create_user(email=email, password=rand_str(14))
+        log = self.uk.logs.log_event(actor='NOT_A_VAlID_USER_ID', actor_ip='::1', actor_useragent='TestLogs',
+                                     actee=user.id, action='test.custom_audit_log', details='Testing custom audit logs')
+        self.assertIsNone(log)
```

Comment:
Are theses tests passing for you? I get an error on the line above. I think we need to assert that the correct error occurs in our test, instead of asserting that `log` is None.